buglist.cgi

```diff
@@ -512,6 +514,9 @@ if (defined $params->param('columnlist')) {
   }
   else {
     @displaycolumns = split(/[ ,]+/, $params->param('columnlist'));
+    # Weed out duplicate column names
+    @displaycolumns = uniq @displaycolumns;
+    $params->param('columnlist', join ',', @displaycolumns);
```

Comment:
Oh, do we both peel out `$params->param` into `@displaycolumns`, then continue to use `$params->param` elsewhere? That's a bit of a bummer :(

Reply:
Yeah unfortunately.

https://github.com/mozilla-bteam/bmo/blob/master/buglist.cgi#L1049-L1051

Bugzilla never one any code design awards :)